docs/tasks/administer-cluster/reconfigure-kubelet.md

```diff
@@ -0,0 +1,264 @@
+---
+approvers:
+- mtaufen
+- dawnchen
+title: Reconfigure a Node's Kubelet in a Live Cluster
+---
+
+{% capture overview %}
+As of Kubernetes 1.8, the new [Dynamic Kubelet Configuration](https://github.com/kubernetes/features/issues/281) feature is available in alpha. This allows you to change the configuration of Kubelets in a live Kubernetes cluster via first-class Kubernetes concepts. Specifically, this feature allows you to configure individual Nodes' Kubelets via ConfigMaps.
+
+**Warning:** The organization of the Kubelet's configuration type is still considered alpha, and it is likely to be restructured with breaking changes before graduating from alpha.
+
+**Warning:** All Kubelet configuration parameters may be changed dynamically, but not all parameters are safe to change dynamically. This feature is intended for system experts who have a strong understanding of how configuration changes will affect behavior. No documentation currently exists which plainly lists "safe to change" fields, but we plan to add it before this feature graduates from alpha.
+{% endcapture %}
+
+{% capture prerequisites %}
+- A live Kubernetes cluster at version 1.8 or higher must be running, with the `DynamicKubeletConfig` feature gate enabled and the Kubelet's `--dynamic-config-dir` flag set to a writeable directory on the Node.
+- The kubectl command-line tool (also v1.8 or higher) must be configured to communicate with the cluster.
+{% endcapture %}
+
+{% capture steps %}
+
+## Reconfiguring the Kubelet on a Live Node in your Cluster
+
+### Basic Workflow Overview
+
+The basic workflow for configuring a Kubelet in a live cluster is as follows:
+
+1. Write a yaml or json configuration file containing the Kubelet's configuration.
+2. Wrap this file in a ConfigMap and save it to the Kubernetes control plane.
+3. Update the Kubelet's correspoinding Node object to use this ConfigMap.
+
+Each Kubelet watches a configuration reference on its respective Node object. When this reference changes, the Kubelet downloads the new configuration and restarts to begin using it. 
+
+The status of the Node's Kubelet configuration is reported via the `ConfigOK` condition in the Node status. Once you have updated a Node to use the new ConfigMap, you can observe this condition to confirm that the Node is using the intended configuration. A table describing the possible conditions can be found at the end of this article.
+
+This document describes editing Nodes using `kubectl edit`. There are other ways to modify a Node's spec, including `kubectl patch`, for example, which facilitate scripted workflows.
+
+This document only describes a single Node consuming each ConfigMap. Keep in mind that it is also valid for multiple Nodes to consume the same ConfigMap.
+
+### Workaround: Add system:Nodes subject to the system:Node ClusterRoleBinding 
```

Comment:
Yeah this was a major facepalm for me, because @liggitt warned me about this and then I realized we hadn't done it when I was testing these docs...

Yeah I can switch to a per-node binding.

Reply:
I plan to add read access to the config map to the node authorizer the same release the field leaves alpha, so probably 1.9

Reply:
Cool, thanks @liggitt 
@mtaufen I think k8s docs should be "best-practice" more than "quick-hack" style :wink:

Reply:
Can't argue with that :)